From 97089654e0a094dcd72c9841f29be12e6fa36d9f Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Wed, 2 Jul 2014 14:50:50 +0800
Subject: [PATCH 337/602] ENGR00242365 Update BUILD_ID for KK4.4.3_2.0.0-Beta
 release

Update BUILD_ID as "2.0.0-beta-rc1"
Update BUILD_NUMBER as "20140806" to align release date

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 arm2_6dq/build_id.mk       | 4 ++--
 arm2_6sl/build_id.mk       | 4 ++--
 evk_6sl/build_id.mk        | 4 ++--
 hdmidongle_6dq/build_id.mk | 4 ++--
 imx5x/build_id.mk          | 4 ++--
 sabreauto_6q/build_id.mk   | 4 ++--
 sabresd_6dq/build_id.mk    | 4 ++--
 sabresd_6sx/build_id.mk    | 4 ++--
 8 files changed, 16 insertions(+), 16 deletions(-)

diff --git a/arm2_6dq/build_id.mk b/arm2_6dq/build_id.mk
index 1548422..a8f7e55 100644
--- a/arm2_6dq/build_id.mk
+++ b/arm2_6dq/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=2.0.0-alpha-rc2
-export BUILD_NUMBER=20140520
+export BUILD_ID=2.0.0-beta-rc1
+export BUILD_NUMBER=20140806
diff --git a/arm2_6sl/build_id.mk b/arm2_6sl/build_id.mk
index 1548422..a8f7e55 100644
--- a/arm2_6sl/build_id.mk
+++ b/arm2_6sl/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=2.0.0-alpha-rc2
-export BUILD_NUMBER=20140520
+export BUILD_ID=2.0.0-beta-rc1
+export BUILD_NUMBER=20140806
diff --git a/evk_6sl/build_id.mk b/evk_6sl/build_id.mk
index 1548422..a8f7e55 100755
--- a/evk_6sl/build_id.mk
+++ b/evk_6sl/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=2.0.0-alpha-rc2
-export BUILD_NUMBER=20140520
+export BUILD_ID=2.0.0-beta-rc1
+export BUILD_NUMBER=20140806
diff --git a/hdmidongle_6dq/build_id.mk b/hdmidongle_6dq/build_id.mk
index 1548422..a8f7e55 100644
--- a/hdmidongle_6dq/build_id.mk
+++ b/hdmidongle_6dq/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=2.0.0-alpha-rc2
-export BUILD_NUMBER=20140520
+export BUILD_ID=2.0.0-beta-rc1
+export BUILD_NUMBER=20140806
diff --git a/imx5x/build_id.mk b/imx5x/build_id.mk
index 1548422..a8f7e55 100644
--- a/imx5x/build_id.mk
+++ b/imx5x/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=2.0.0-alpha-rc2
-export BUILD_NUMBER=20140520
+export BUILD_ID=2.0.0-beta-rc1
+export BUILD_NUMBER=20140806
diff --git a/sabreauto_6q/build_id.mk b/sabreauto_6q/build_id.mk
index 1548422..a8f7e55 100644
--- a/sabreauto_6q/build_id.mk
+++ b/sabreauto_6q/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=2.0.0-alpha-rc2
-export BUILD_NUMBER=20140520
+export BUILD_ID=2.0.0-beta-rc1
+export BUILD_NUMBER=20140806
diff --git a/sabresd_6dq/build_id.mk b/sabresd_6dq/build_id.mk
index 1548422..a8f7e55 100644
--- a/sabresd_6dq/build_id.mk
+++ b/sabresd_6dq/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=2.0.0-alpha-rc2
-export BUILD_NUMBER=20140520
+export BUILD_ID=2.0.0-beta-rc1
+export BUILD_NUMBER=20140806
diff --git a/sabresd_6sx/build_id.mk b/sabresd_6sx/build_id.mk
index 1548422..a8f7e55 100644
--- a/sabresd_6sx/build_id.mk
+++ b/sabresd_6sx/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=2.0.0-alpha-rc2
-export BUILD_NUMBER=20140520
+export BUILD_ID=2.0.0-beta-rc1
+export BUILD_NUMBER=20140806
-- 
1.8.0

